client/ayon_core/pipeline/create/creator_plugins.py

```diff
@@ -362,6 +363,18 @@ def log(self):
             self._log = Logger.get_logger(self.__class__.__name__)
         return self._log
 
+    def _create_instance(
+        self, product_type: str, product_name: str, data: Dict[str, Any]
+    ) -> CreatedInstance:
+        instance = CreatedInstance(
+            product_type,
+            product_name,
+            data,
+            creator=self,
+        )
+        self._add_instance_to_context(instance)
+        return instance
```

Comment:
Why is this needed?

Reply:
Just helper method for creators. Not needed, but you don't have to import `CreatedInstance` to create one.